cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py

```diff
@@ -0,0 +1,145 @@
+    """
+    separatrix_SOL_collisionality = calc_separatrix_SOL_collisionality(
+        separatrix_electron_density=separatrix_electron_density,
+        separatrix_electron_temp=separatrix_electron_temp,
+        parallel_connection_length=parallel_connection_length,
+    )
+
+    Spitzer_conduction_reduction_factor = calc_Spitzer_conduction_reduction_factor_scaling(
+        separatrix_SOL_collisionality=separatrix_SOL_collisionality,
+    )
+
+    delta_electron_sheath_factor = calc_delta_electron_sheath_factor(
+        separatrix_electron_temp=separatrix_electron_temp,
+        target_electron_temp=target_electron_temp,
+        SOL_momentum_loss_fraction=SOL_momentum_loss_fraction,
+    )
+    return separatrix_SOL_collisionality, Spitzer_conduction_reduction_factor, delta_electron_sheath_factor
+
+
+def calc_Spitzer_conduction_reduction_factor_scaling(
+    separatrix_SOL_collisionality: Unitfull,
+) -> Union[float, xr.DataArray]:
+    """Factor to reduce the electron heat conduction (relative to Braginskii) due to kinetic effects, according to SOL-KiT.
+
+    Equation 10 from :cite:`Power_2023`
+
+    Args:
+        separatrix_SOL_collisionality: [~] :term:`glossary_link<separatrix_SOL_collisionality>`
+
+    Returns:
+        :term:`Spitzer_conduction_reduction_factor` [~]
+    """
+    Spitzer_conduction_reduction_factor = 0.696 * np.exp(-8.059 * (separatrix_SOL_collisionality) ** -1.074) + 0.260
+    Spitzer_conduction_reduction_factor = xr.where(
+        Spitzer_conduction_reduction_factor < 0.5, 1.0 / (1 + 6.584 / separatrix_SOL_collisionality), Spitzer_conduction_reduction_factor
+    )
+
+    return Spitzer_conduction_reduction_factor
+
+
+@wraps_ufunc(
+    return_units=dict(Spitzer_conduction_reduction_factor=ureg.dimensionless),
+    input_units=dict(
+        separatrix_electron_density=ureg.m**-3,
+        separatrix_electron_temp=ureg.eV,
+        parallel_connection_length=ureg.m,
+        target_electron_temp=ureg.eV,
+        kappa_e0=ureg.W / (ureg.eV**3.5 * ureg.m),
+        electron_mass=ureg.kg,  # TODO: electron mass in kg?
+        electron_charge=ureg.C,  # TODO: elementary charge in Coulomb?
+        SOL_conduction_fraction=ureg.dimensionless,
+        flux_limit_factor_alpha=ureg.dimensionless,
+    ),
+)
+def calc_Spitzer_conduction_reduction_factor_fluxlim(
+    separatrix_electron_density: Unitfull,
+    separatrix_electron_temp: Unitfull,
+    parallel_connection_length: Unitfull,
+    target_electron_temp: Unitfull,
+    kappa_e0: Unitfull,
+    electron_mass: Unitfull = 1.0 * ureg.electron_mass,
+    electron_charge: Unitfull = 1.0 * ureg.elementary_charge,
+    SOL_conduction_fraction: Union[float, xr.DataArray] = 1.0,
+    flux_limit_factor_alpha: Union[float, xr.DataArray] = 0.15,
+) -> Union[float, xr.DataArray]:
+    """Factor to reduce the electron heat conduction (relative to Braginskii) due to kinetic effects, via a flux limiter.
+
+    #TODO: Needs reference and units need to be checked
```

Comment:
@mwigram

Reply:
Reference: "The Plasma Boundary of Magnetic Fusion Devices", Peter C Stangeby, Chapter 26

Units are correct